certbot-apache/certbot_apache/interfaces.py

```diff
@@ -131,9 +131,10 @@ class ParserNode(object):
 
     ParserNode objects should have the following attributes:
 
-    # Reference to ancestor node, or None if the node is the root node of the
-    # configuration tree.
-    ancestor: Optional[ParserNode]
+    # Reference to ancestor node(s), empty if the node is the root node of the
+    # configuration tree. This node can have multiple ancestors in the case
+    # that it is the root of a file included by multiple files.
```

Comment:
We're using `root` twice here, and as I understand it, it doesn't have quite the same meaning both times. Could we find a different word for the second usage?